curvefs/src/client/rpcclient/metaserver_client.cpp

```diff
@@ -497,163 +497,185 @@ bool GroupInodeIdByPartition(
     return true;
 }
 
+bool MetaServerClientImpl::SplitRequestInodes(
+    uint32_t fsId,
+    const std::set<uint64_t> &inodeIds,
+    std::vector<std::list<uint64_t>> *inodeGroups) {
+    std::unordered_map<uint32_t, std::list<uint64_t>> groups;
```

Comment:
for most cases, `std::vector` is always preferred, std::list/map/set are all node-based containers, they're not cache-friendly.

for `std::vector`, another tip is if you know the number of elements that it will store, you can call `vec.reserve` firstly to allocate enough memory.

Reply:
fixed